src/compute/src/sink/copy_to_s3_oneshot.rs

```diff
@@ -67,68 +81,272 @@ where
 }
 
 fn copy_to<G>(
```

Comment:
According to the [coverage data](https://buildkite.com/materialize/coverage/builds/372), this function is not covered by any test case. Could you add a test in a follow-up PR?

Reply:
OOC does nightly ci count as part of coverage?

Reply:
No, it does not run with coverage.